src/EditorFeatures/Core/Implementation/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -41,7 +41,7 @@ public SearchResult(Document document, DeclaredSymbolInfo declaredSymbolInfo, st
                 var declaredNavigableItem = navigableItem as NavigableItemFactory.DeclaredSymbolNavigableItem;
                 Debug.Assert(declaredNavigableItem != null);
 
-                _lazySummary = new Lazy<string>(() => declaredNavigableItem.Symbol.GetDocumentationComment()?.SummaryText);
+                _lazySummary = new Lazy<string>(() => declaredNavigableItem.Symbol?.GetDocumentationComment()?.SummaryText);
```

Comment:
Ultimately, we're calling through SemanticModel.GetDeclaredSymbolForNode() for which null is a valid return value.  This null check isn't directly related to cancellation, but I just spotted a potential future NRE when checking all references to DeclaredSymbolNavigableItem.Symbol that I wanted to prevent (especially since we're guarding against another NRE before descending into SummaryText).